### PR TITLE
refactor(content): update Cloudflare and Watchtower info

### PR DESCRIPTION
**Pull request by suprovsky**

### CODE REVIEW

**Review by garrappachc — APPROVED** [review]

File: .github/workflows/build-docker-image.yml [hunk 1]
```diff
@@ -15,14 +15,14 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
       - name: Login to GitHub Container Registry
-        uses: docker/login-action@v2
```
Comment (garrappachc) [inline .github/workflows/build-docker-image.yml:18]: Please make these changes in separate PRs

Comment (suprovsky) [reply .github/workflows/build-docker-image.yml:18]: Could I start doing it from the next time please? I just want to have it passed and in operation.
